pkg/vsphere/doc.go

```diff
@@ -6,6 +6,6 @@ SPDX-License-Identifier: Apache-2.0
 // Package vsphere holds utilities for bootstrapping a vSphere API client
 // from the metadata injected by the VSphereSource.  Within a receive adapter,
 // users can write:
-//    client, err := vsphere.New(ctx)
+//    client, err := vsphere.NewSOAPClient(ctx)
```

Comment:
I wonder if we should add something here about the need to perform client.Logout() after work has been finished?

Reply:
Done